docs/topics/api/overview.rst

```diff
@@ -412,7 +412,8 @@ These are `v5` specific changes - `v4` changes apply also.
 * 2021-02-04: dropped /shelves/sponsored endpoint https://github.com/mozilla/addons-server/issues/16390
 * 2021-02-11: removed Stripe webhook endpoint https://github.com/mozilla/addons-server/issues/16391
 * 2021-02-11: added ``show_grouped_ratings`` to addon detail endpoint. https://github.com/mozilla/addons-server/issues/16459
-* 2021-02-12: added ``versions_url`` to addon detail endpoint. https://github.com/mozilla/addons-server/issues/16534
+* 2021-02-18: made ``description``, ``headline``, and ``cta.text`` in shelves/hero endpoint translated fields in the response. (They were always localized, we just didn't return them as such). https://github.com/mozilla/addons-server/issues/16515
+* 2021-02-18: added ``versions_url`` to addon detail endpoint. https://github.com/mozilla/addons-server/issues/16534
```

Comment:
Why did you change the date?

Reply:
we usually date the change for the production push as that's when it'll be available for people to use.  (it's not a strict rule, I just changed it because it was the next line)